src/components/base/outline-button/outline-button.css

```diff
@@ -1,5 +1,11 @@
+:host {
+  /* flex and border-radius constrain the actual boundaries of the element,
+   preventing literal edge cases where a user can click on the edge or corner of the button and defeate states like disabled */
+  @apply flex rounded-xl;
+}
+
 .btn {
-  @apply px-12 py-4 text-lg leading-tight rounded-xl cursor-pointer transition-colors duration-300 font-body no-underline inline-flex content-center;
```

Comment:
Can we add the padding back here. I thought `py-3 px-4` looked pretty good.

Reply:
It's still there for large buttons.